### PR TITLE
Fixed actual default collation representative

### PR DESCRIPTION
**Pull request by Hydrocharged**

### CODE REVIEW

File: sql/charsetcollation.go [hunk 1]
```diff
@@ -709,9 +707,278 @@ type mysqlCollationRow struct {
 }
 
 var CollationToMySQLVals = map[string]mysqlCollationRow{
-	Collation_binary.Name:             {63, Y, Y, 0, NoPad},
-	Collation_utf8_general_ci.Name:    {33, Y, Y, 1, PadSpace},
-	Collation_utf8mb4_0900_ai_ci.Name: {255, Y, Y, 0, NoPad},
+	Collation_armscii8_general_ci.Name:         {32, Y, Y, 1, PadSpace},
```
Comment (zachmu) [inline sql/charsetcollation.go:710]: Verified each one of these by hand ;)

File: sql/parse/parse.go [hunk 2]
```diff
@@ -525,6 +525,19 @@ func convertShow(ctx *sql.Context, s *sqlparser.Show, query string) (sql.Node, e
 			if err != nil {
 				return nil, err
 			}
+			// TODO: once collations are properly implemented, we should better be able to handle utf8 -> utf8mb3 comparisons as they're aliases
+			filterExpr, err = expression.TransformUp(ctx, filterExpr, func(expr sql.Expression) (sql.Expression, error) {
```
Comment (zachmu) [inline sql/parse/parse.go:529]: engine test for this?

**Review by zachmu — APPROVED** [review]
LG, just one small comment

File: sql/parse/parse.go (hunk 2 above)
Comment (Hydrocharged) [reply sql/parse/parse.go:529]: This is caught by one of the modified engine tests, in particular the `IN (<collations>)` one.

**Pull request merged**